index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Henary's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div >
+            <h1 class="brand">My Logo</h1>
+        </div>
+        <nav>
+            <img class="menuIcon" src="images/menuIcon.png" alt=" menu">
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <h2 class="introduce">I’m Henary
+                Glad to see you!</h2>
+             <p class="discription">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             <p class="contact">LET’S CONNECT</p>
+
+             <ul class="social-connection"> 
+                <img src="images/twiter.png" alt="twiter">
+                <img src="images/Linkedin icon.png" alt="Linkedin">
+                <img src="images/M.png" alt="M">
+                <img src="images/V.png" alt="V">
+            
```

Comment:
Good job using a list `ul` element however, i suggest you change the children  to be `li`. Make it a `<li><a><img>`. This ideally should be links to your socials and using link tags lets you use `href` to connect to them.